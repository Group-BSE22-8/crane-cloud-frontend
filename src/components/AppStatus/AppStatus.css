.StatusWrapper {
  width: 100%;
  display: grid;
  justify-content: center;
}
.StatusSignal {
  border-radius: 50%;
  width: 0.8rem;
  height: 0.8rem;
}

.StatusRunning {
  background-color: #98a058;
}

.StatusFailed {
  background-color: #ff0000;
}

.StatusOther {
  background-color: var(--secondary-color);
}
