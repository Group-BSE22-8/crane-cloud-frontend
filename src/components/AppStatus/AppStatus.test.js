import React from 'react';

import { shallow } from 'enzyme';

import AppStatus from './index';

describe('AppStatus Component', () => {
  it('Renders a AppStatus component', () => {
    const AppStatusComponent = shallow(<AppStatus/>);
    expect(AppStatusComponent.find('.StatusWrapper').hasClass('StatusWrapper')).toBe(true);
   expect(AppStatusComponent).toMatchSnapshot();
  });
 
});