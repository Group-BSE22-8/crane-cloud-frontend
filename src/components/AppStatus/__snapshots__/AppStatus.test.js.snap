// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppStatus Component Renders a AppStatus component 1`] = `ShallowWrapper {}`;
